### DISCUSSION TITLE
Add National Levee Database to PyGeoHydro

### DISCUSSION THREAD

**Discussion by colby-thrash** [hyriver/pygeohydro#117] [Ideas]
Would you ever consider adding the USACE [National Levee Database](https://levees.sec.usace.army.mil/) to PyGeoHydro Web services? I see there is a levee layer in the NFHL module but it seems to only contain a few of the levees from my one test area.

Thanks for creating these excellent tools!

**Comment by cheginit** [comment]
Thanks for the suggestions, I will look into it. Last year I checked their web service, and implementing it wasn't very straightforward. So, I will look into again, hopefully, it's been improved!

**Reply by cheginit** [reply]
@colby-thrash It took a while for me to get around to this.

I just added support their REST service. Can you please give it a try and let me know if it contains the data that is usually needed for common levee analyses? You can install `pygeohydro` from git and then just do `from pygeohydro import NLD`. The class's docstring has an example.

**Reply by colby-thrash** [reply]
@cheginit Awesome thanks for looking into this! Sorry it has taken me a while to get back to this as well. I installed from git and plan to take a look this week.
